### PR TITLE
[DBAAS-940] add CATALOG_BASE_IMG to ensure seamless upgrade

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: Tommy Hughes <tohughes@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -215,7 +215,7 @@ ifeq (,$(shell which opm 2>/dev/null))
 	set -e ;\
 	mkdir -p $(dir $(OPM)) ;\
 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.19.1/$${OS}-$${ARCH}-opm ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.23.2/$${OS}-$${ARCH}-opm ;\
```
Comment (redhatHameed) [inline Makefile:218]: why opm version change ?

File: Makefile [hunk 2]
```diff
@@ -14,7 +14,7 @@ RDS_VERSION ?= 0.0.27
 ORG ?= ecosystem-appeng
 
 # CATALOG_BASE_IMG defines an existing catalog version to build on & add bundles to
-#CATALOG_BASE_IMG ?= quay.io/$(ORG)/rds-dbaas-operator-catalog:v$(VERSION)
+CATALOG_BASE_IMG ?= quay.io/ecosystem-appeng/rds-dbaas-operator-catalog:v0.1.0
```
Comment (redhatHameed) [inline Makefile:17]: why sticking with v0.1.0 ?

File: Makefile (hunk 1 above)
Comment (tchughesiv) [reply Makefile:218]: thats the version we use for dbaas-operator... also, the old version was throwing errors for me

File: Makefile (hunk 2 above)
Comment (tchughesiv) [reply Makefile:17]: because that's the prior release we're going to build this release on top of

Comment (redhatHameed) [reply Makefile:17]: I see the operator docker image version is different nvm

**Review by redhatHameed — APPROVED** [review]
LGTM

**Pull request merged**